manim/mobject/geometry.py

```diff
@@ -739,15 +760,27 @@ def __init__(
         Line.__init__(self, *args, **kwargs)
         dashes = DashedVMobject(
             self,
-            num_dashes=self.calculate_num_dashes(positive_space_ratio),
+            num_dashes=self.calculate_num_dashes(),
             positive_space_ratio=positive_space_ratio,
         )
         self.clear_points()
         self.add(*dashes)
 
-    def calculate_num_dashes(self, positive_space_ratio):
+    def calculate_num_dashes(self):
+        """Returns the number of dashes in the dashed line.
+
+        Examples
+        --------
+        ::
+
+            >>> DashedLine().calculate_num_dashes()
+            20
+
+        :class:`int`
```

Comment:
Personally, I would remove the return type from the docstring, and rather add a type hint to the method header: `def calculate_num_dashes(self) -> int:`.

```suggestion
```